README.md

```diff
@@ -6,69 +6,85 @@ Utilities for interacting with a TPM2.0 used for access to the Xaptum Edge Netwo
 
 [![Build Status](https://travis-ci.org/xaptum/xaptum-tpm.svg?branch=master)](https://travis-ci.org/xaptum/xaptum-tpm)
 
-## Requirements
-- cmake version >= 3.0
-- A C99-compliant compiler
-- Currently, only supports POSIX platforms
+## Installation
 
-## Building
+`xaptum-tpm` is available for the following distributions. It may also
+be built from source.
 
-`xaptum-tpm` uses CMake as its build system:
+### Debian Jessie
 
 ```bash
-# Create a subdirectory to hold the build
-mkdir -p build
-cd build
+# Add the repository to APT sources
+echo "deb http://dl.bintray.com/xaptum/deb jessie main" | sudo tee -a /etc/apt/sources.list
```

Comment:
Running `apt-key` requires super-user, and since we're including the `sudo` in the `tee` above, should we explicitly include it for the `apt-key`, too?

Reply:
Yeah, probably.  I'd only included it for the first because that's the only tricky invocation (needing `sudo tee -a`), but probably better to be consistent.